src/types.ts

```diff
@@ -94,32 +94,23 @@ export interface IdPServerResponse {
 }
 
 /**
- * A copy of the Node.js driver's `OIDCCallbackContext`
+ * A copy of the Node.js driver's `OIDCCallbackParams` using `OIDCAbortSignal` instead of `AbortSignal`
  * @public
  */
-export interface OIDCCallbackContext {
+export interface OIDCCallbackParams {
   refreshToken?: string;
-  timeoutSeconds?: number;
-  timeoutContext?: OIDCAbortSignal;
-  version: number;
+  timeoutContext: OIDCAbortSignal;
```

Comment:
```suggestion
  timeoutContext?: OIDCAbortSignal;
```

Or is there a specific reason why this needs to be required?

Reply:
driver has marked it as required prop. if we don't have to align this bit, i'll leave it as optional

Reply:
We can always be more accepting than the driver